src/gui/src/layoutViewer.cpp

```diff
@@ -3263,6 +3263,17 @@ void LayoutViewer::selectHighlightConnectedNets(bool select_flag,
       select_flag, output, input, highlight_group);
 }
 
+void LayoutViewer::selectHighlightConnectedBufferTrees(bool select_flag)
+{
+  int highlight_group = 0;
+  if (!select_flag) {
+    HighlightGroupDialog dlg;
+    dlg.exec();
+    highlight_group = dlg.getSelectedHighlightGroup();
```

Comment:
Why do you need to use a dialog for this?  @gadfort is this the best way?

Reply:
In this case it might be easier to have a submenu that gives you the different highlight groups.

Reply:
Did you mean only in buffer tree or in all the highlight options?